plasmapy/formulary/distribution.py

```diff
@@ -142,8 +146,16 @@ def Maxwellian_1D(v, T, particle="e", v_drift=0, vTh=np.nan, units="units"):
         return distFunc
 
 
+@particle_input
 def Maxwellian_velocity_2D(
-    vx, vy, T, particle="e", vx_drift=0, vy_drift=0, vTh=np.nan, units="units"
+    vx,
+    vy,
+    T,
+    particle: ParticleLike = "e",
+    vx_drift=0,
+    vy_drift=0,
+    vTh=np.nan,
+    units="units",
```

Comment:
I had to do a scroll-up-double-take at what the units argument meant here. Just now realizing that it might be too much customization and we should probably validate the units initially (O(1)), force unitless and append them at the end. Obviously out of scope for this PR, but I'll make an issue out of this.

Reply:
I'm wondering if the parameter should be `skip_validations` instead of `units`.  In any case, that's for another day!